### PR TITLE
feat/34254 Basic Layout structure for DAO Proposals form

### PR DESCRIPTION
**Pull request by roshan-bisht-lab49**
Basic Layout for Create DAO Proposals form

Screenshot 2023-06-05 at 5 50 10 PM

### CODE REVIEW

**Review by roshan-bisht-lab49 — COMMENTED** [review]
code comments

**Comment by roshan-bisht-lab49** [comment]
Few UI Notes:
1. The DAOType Form in Figma was not under the stepper but I guess it is the part of the Wizard form code wise also it matches the exact UI layout let me know if this is okay or do we need to remove the DAOTypeForm from stepper steps.

**Review by damiano-melcarne-lab49 — APPROVED** [review]

**Comment by damiano-melcarne-lab49** [comment]
Yes, the type selector should be under the stepper wizard (`Type`).

File: src/dex-ui/pages/dao/DAOProposals/Forms/DAODeleteMemberReviewForm.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          <Flex gap="2" alignItems="center">
+            <Text textStyle="p small regular" color={Color.Neutral._700}>
+              {memberAddress}
+            </Text>
+            <CopyTextButton onClick={handleCopyMemberId} iconSize="17" />
+          </Flex>
+        </Flex>
+        <Divider />
+        <Flex direction="row" gap="2" alignItems="center">
+          <Text textStyle="p small medium">Transaction threshold confirmation changes to</Text>
```
Comment (damiano-melcarne-lab49) [inline src/dex-ui/pages/dao/DAOProposals/Forms/DAODeleteMemberReviewForm.tsx:47]: `Transaction` -> `Proposal`

File: src/dex-ui/pages/dao/DAOProposals/Forms/DAOAddMemberDetailsForm.tsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+import { DAOFormContainer } from "../../CreateADAO/forms/DAOFormContainer";
+
+export function DAOAddMemberDetailsForm() {
+  const { threshold, membersCount } = useOutletContext<CreateDAOProposalContext>();
+  const {
+    register,
+    formState: { errors },
+  } = useFormContext<CreateDAOMemberOperationForm>();
+  return (
+    <DAOFormContainer>
```
[the remark this answers is not in the document]
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/DAOProposals/Forms/DAOAddMemberDetailsForm.tsx]: Done

File: src/dex-ui/pages/dao/DAOProposals/Forms/DAOAddMemberDetailsForm.tsx [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+              }),
+            },
+          }}
+          isInvalid={Boolean(errors?.memberAddress)}
+          errorMessage={errors?.memberAddress && errors?.memberAddress?.message}
+        />
+        <Flex direction="column" gap="1">
+          <Flex direction="row" gap="1" alignItems="center">
+            <Text textStyle="p medium regular" color={Color.Neutral._500}>
+              Transaction threshold confirmation requirement is
```
[the remark this answers is not in the document]
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/DAOProposals/Forms/DAOAddMemberDetailsForm.tsx]: Done

File: src/dex-ui/pages/dao/DAOProposals/Forms/DAODeleteMemberReviewForm.tsx (hunk 1 above)
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/DAOProposals/Forms/DAODeleteMemberReviewForm.tsx]: Done

File: src/dex-ui/pages/dao/DAOProposals/Forms/DAOUpgradeThresholdDetailsForm.tsx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+                validate: (value) => value.length <= 240 || "Maximum character count for the description is 240.",
+              }),
+            },
+          }}
+          isInvalid={Boolean(errors?.description)}
+          errorMessage={errors?.description && errors?.description?.message}
+        />
+        <Flex direction="row" gap="1" alignItems="center">
+          <Text textStyle="p medium regular" color={Color.Neutral._500}>
+            Current transaction threshold confirmation requirement is
```
[the remark this answers is not in the document]
Comment (roshan-bisht-lab49) [reply src/dex-ui/pages/dao/DAOProposals/Forms/DAOUpgradeThresholdDetailsForm.tsx]: Done

**Review by damiano-melcarne-lab49 — APPROVED** [review]

**Pull request merged**